source/filter.js

```diff
@@ -44,8 +45,10 @@ var filter = _curry2(_dispatchable(['fantasy-land/filter', 'filter'], _xfilter,
         }
         return acc;
       }, {}, keys(filterable)) :
-    // else
-      _filter(pred, filterable)
+      _isMap(filterable) ?
```

Comment:
Do we need a new internal function for this?

```suggestion
      is(Map, filterable) ?
```

Reply:
@Harris-Miller your choice 😁 , I was just replicating
https://github.com/ramda/ramda/blob/fdf8722d6cde38c4f8f85cc557100f83f431fd81/source/internal/_isObject.js#L1

Reply:
Looking at this move, yes let's keep `_isMap` as you have it